docs/rest_api/endpoints/models/optimize_model_endpoint.yaml

```diff
@@ -13,12 +13,14 @@ post:
     - $ref: '../../parameters/path/model_group_id.yaml'
     - $ref: '../../parameters/path/model_id.yaml'
   requestBody:
+    required: false
     content:
       application/json:
         schema:
-          $ref: "../../../../interactive_ai/services/api/schemas/models/requests/optimization_request.yaml"
+          type: object
+          additionalProperties: false
         examples:
-          Simple optimize request:
+          Empty optimize request:
```

Comment:
You can remove this example too. It's empty, so the example doesn't provide any value.